metagraph/src/graph/representation/canonical_dbg.cpp

```diff
@@ -211,7 +218,9 @@ ::call_incoming_kmers(node_index node, const IncomingEdgeCallback &callback) con
         std::vector<node_index> parents(alph_size);
 
         graph_.call_incoming_kmers(node, [&](node_index prev, char c) {
-            parents[alph_map_[c]] = prev;
+            if (c != boss::BOSS::kSentinel)
+                parents[alph_map_[c]] = prev;
+
             --alph_size;
```

Comment:
What does this alph_size refer to? Better introduce another variable and pick an appropriate name for it.

Reply:
I renamed it to `max_num_edges_left`. It's basically used to check whether we should bother to check the incoming nodes of the reverse complement.